client/src/components/complex-components/NavBar/GraveNavBar.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
```

Comment:
Why did you copy and paste this component? What's the difference between this NavBar and the original?

You should be extending the original NavBar and passing a prop to change the styling so it matches your needs for this page. This will be a pain to manage in the future because now any updates we do to the NavBar would need to be copied over.

Reply:
you so clever i didnt even think of that